subsys/bluetooth/host/hci_core.c

```diff
@@ -2007,8 +2008,11 @@ static void hci_cmd_done(uint16_t opcode, uint8_t status, struct net_buf *buf)
 	if (cmd(buf)->opcode != opcode) {
 		BT_WARN("OpCode 0x%04x completed instead of expected 0x%04x",
 			opcode, cmd(buf)->opcode);
+		return;
```

Comment:
This is fixing b) in https://github.com/zephyrproject-rtos/zephyr/issues/38054#issuecomment-911518510